e2e/test/scenarios/onboarding/embedding-homepage.cy.spec.ts

```diff
@@ -29,37 +28,14 @@ describeWithSnowplow("scenarios > embedding-homepage > snowplow events", () => {
     expectNoBadSnowplowEvents();
   });
 
-  it("clicking on the quickstart button should send the 'embedding_homepage_quickstart_click' event", () => {
```

Comment:
I'm waiting for alberto to confirm, but I don't think we need this event anymore, not in this shape at least